project/core/templates/base.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n %}
 {% load static %}
-<!DOCTYPE html>
+<!DOCTYPE HTML>
```

Comment:
This should remain `<!DOCTYPE html>`: https://developer.mozilla.org/en-US/docs/Glossary/Doctype
```suggestion
<!DOCTYPE html>
```